src/globals.c

```diff
@@ -559,6 +559,7 @@ static const struct option global_opts[] = {
 	{ "time", no_argument, 0, 't' },
 	{ "url", required_argument, 0, 'u' },
 	{ "versionurl", required_argument, 0, 'v' },
+	{ "max-parallel-downloads", required_argument, 0, 'D' },
```

Comment:
Maybe add a TODO here with the comment that D was deprecated and we should remove this option from the base code in X months